YubiKit/YubiKitTests/Tests/YKFTestCase.m

```diff
@@ -54,21 +55,4 @@ + (NSData *)dataFromHexString:(NSString *)string {
     return data;
 }
 
-- (NSString *)ykf_hexadecimalString
-{
-    /* Returns hexadecimal string of NSData. Empty string if data is empty. */
-    const unsigned char *dataBuffer = (const unsigned char *)[self bytes];
-    if (!dataBuffer) {
-        return [NSString string];
-    }
-
-    NSUInteger          dataLength  = [self length];
-    NSMutableString     *hexString  = [NSMutableString stringWithCapacity:(dataLength * 2)];
-    for (int i = 0; i < dataLength; ++i) {
-        [hexString appendFormat:@"%02x", (unsigned int)dataBuffer[i]];
```

Comment:
cast should be unnecessary (and best not to do if not needed to avoid future errors, in general), at least it would in C

Reply:
That code has been removed.